packages/terra-graphs-docs/src/terra-dev-site/test/graphs/Combination/BasicCombinationGraphs/LineWithSingleBubbleData.test.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import CombinationGraph from '@cerner/terra-graphs/lib/components/Combination/CombinationGraph';
+import utils from '@cerner/carbon-graphs/lib/js/helpers/utils';
+import '@cerner/terra-graphs-docs/lib/Css/ExampleGraphContainer.module.scss';
+import getCombinationGraphConfig from '@cerner/terra-graphs-docs/lib/example-datasets/graphConfigObjects/Combination/combinationDefault';
+import data from '@cerner/terra-graphs-docs/lib/example-datasets/dataObjects/Combination/lineWithSingleBubbleData';
+
+const graphConfig = utils.deepClone(getCombinationGraphConfig('#lineWithPairedResult'));
+graphConfig.axis.x.ticks = {
+  values: [1, 2, 3, 4, 5, 6, 7],
+  format: '.0f',
+};
+graphConfig.showVGrid = false;
+graphConfig.axis.y2.show = true;
+graphConfig.axis.y2.padDomain = false;
+
+const dataset = utils.deepClone(data);
+
+export default () => (
+  <React.Fragment>
+    <div id="tooltip" className="initial-tooltip" />
+    <CombinationGraph graphID="lineWithPairedResult" graphConfig={graphConfig} dataset={dataset} />
```

Comment:
Should the graphID be `lineWithSingleBubbleData` and `#lineWithSingleBubbleData` on line 8?

Reply:
Fixed [here](https://github.com/cerner/terra-graphs/pull/150/commits/768d7923e4f325eaef40c8caa6ff6753069fc96e?short_path=4843e5a#diff-2096bc62d718a746d35ae639edf8ea359ac524477a61f8598a5cb8c10975dd67R17).